app/Transformers/TextTransformer.php

```diff
@@ -106,13 +106,13 @@ public function transformForV2($text)
             default:
                 return [
                     'book_name'        => (string) $text->book_name,
-                    'book_id'          => (string) $text->osis_id,
+                    'book_id'          => (string) $text->book_name,
```

Comment:
Correct me if I am wrong, this will give you the book_name instead of the id?

Reply:
@mgonzalezs see my response above for a complete explanation. In short, we need to match the response from DBP2 for this endpoint. I'm open to pulling in another field that would represent the DBP2 version of the book id, if there is one.